README.md

```diff
@@ -57,41 +56,27 @@ The use of different shapes and styles is supported.
 
 ```javascript
 viewer.addCylinder({start:{x:0.0,y:2.0,z:0.0},
-    				end:{x:0.0,y:10.0,z:0.0},
-                    radius:0.5,
-                    fromCap:false,
-                    toCap:true,
-                    color:'teal'});
+    end:{x:0.0,y:10.0,z:0.0},
+    radius:0.5,
+    fromCap:false,
+    toCap:true,
+    color:'teal'});
 viewer.addCylinder({start:{x:15.0,y:0.0,z:0.0},
-                    end:{x:20.0,y:0.0,z:0.0},
-                    radius:1.0,
-                    color:'black',
-                    fromCap:false,
-                    toCap:false});
-
+    end:{x:20.0,y:0.0,z:0.0},
+    radius:1.0,
+    color:'black',
+    fromCap:false,
+    toCap:false});
 ```
 
-## Building
```

Comment:
Why was this section removed?

Reply:
Because it exists in the documentation in a better form. It is always better to keep things DRY, in code but also in documentation.

Reply:
Because it exists in the documentation in a better form. It is always better to keep things DRY, in code as in documentation.